zion/src/test/scala/edu/uci/ics/cloudberry/zion/actor/DataStoreManagerTest.scala

```diff
@@ -166,14 +166,15 @@ class DataStoreManagerTest extends TestkitExample with SpecificationLike with Mo
       newInfo.stats.lastModifyTime.getMillis must be_>=(now.getMillis)
     }
     "update meta info if receive drop request" in {
+      // this test case is for ???
```

Comment:
if a dataset is dropped, the metadata should not contain the corresponding record.